app.js

```diff
@@ -18,25 +18,29 @@ require("dotenv/config");
 // Import Routes
 const postsRoute = require("./routes/postings");
 const usersRoute = require("./routes/users");
+const notificationsRoute = require("./routes/notifications");
 
 app.use(cors());
-app.use(bodyParser.json({limit: '50mb'}));
+app.use(bodyParser.json({ limit: "50mb" }));
 app.use("/api/postings", postsRoute);
 app.use("/api/users", usersRoute);
+app.use("/api/notifications", notificationsRoute);
```

Comment:
nit: If notifications are user oriented, I wonder if it makes more sense to add it to the users endpoint path e.g /api/users/userId/notifications

Reply:
I think its ok to keep it separate for now but yeah i guess i can change the endpoint but i dont wanna make userRouter become a massive router. The separation right now is fairly clean